### PR TITLE
fix: document & comment typo

### PR DESCRIPTION
**Pull request by tsuyoshicho**
fix below

In comment
- System.Process.System

In document
- ConcurrentProcess
- Text.LTSV

### CODE REVIEW

File: autoload/vital/__vital__/System/Process/System.vim [hunk 1]
```diff
@@ -41,7 +41,7 @@ endfunction
 function! s:execute(args, options) abort
   " NOTE:
   " execute() is a command for executing program WITHOUT using shell.
-  " so mimic that behaviour with shell
+  " so mimic that behavior with shell
```
Comment (ujihisa) [inline autoload/vital/__vital__/System/Process/System.vim:44]: behaviour is correct, so no need to fix

File: doc/vital/Text/LTSV.txt [hunk 2]
```diff
@@ -14,7 +14,7 @@ INTERFACE			|Vital.Text.LTSV-interface|
 ==============================================================================
 INTRODUCTION			*Vital.Text.LTSV-introduction*
 
-*Vital.Text.LTSV* is a parser and dumper for Labelled Tab-separated
+*Vital.Text.LTSV* is a parser and dumper for Labeled Tab-separated
```
Comment (ujihisa) [inline doc/vital/Text/LTSV.txt:17]: labelled is correct too.

File: autoload/vital/__vital__/System/Process/System.vim (hunk 1 above)
Comment (tsuyoshicho) [reply autoload/vital/__vital__/System/Process/System.vim:44]: Thanks, I think other resolve method.

**Comment by tsuyoshicho** [comment]
This PR is closed.
And fix misspell review option change. (US/UK dual OK)

**Pull request closed without merging**